include/pmacc/eventSystem/Manager.tpp

```diff
@@ -73,8 +74,9 @@ namespace pmacc
             if(counter == 500000)
                 std::cout << taskPtr->toString() << " " << passiveTasks.size() << std::endl;
 #endif
-            if(taskPtr->execute())
+            if(taskPtr->execute()) // Looks like the only place in code calling taskPtr->execute().
             {
+		Performance::Timers::append(taskPtr->toString(), taskPtr->perfInfo);
```

Comment:
`taskPtr->toString()` is not unique, this could be the reason for the strange measurements you reported in slack.

Reply:
Indeed - there is only one "TaskKernel" in the output (called 155850 times!).  Each time measurement provides only an upper bound on task time.  If each of the measurements were off by 1 ms, then this would add a "fake" 155 seconds to this task time.  I observed this, since the task times sum to 10x larger than the wall-clock program runtime.